packages/instrumentation-amqplib/test/amqplib-callbacks.spec.ts

```diff
@@ -0,0 +1,131 @@
+import 'mocha';
+import expect from 'expect';
+import sinon from 'sinon';
+import lodash from 'lodash';
```

Comment:
can be removed

Reply:
good catch removed it.
Maybe we should add some linter to the repo one day to catch these issues